features/logging/cluster_log_forwarder.feature

```diff
@@ -495,3 +495,58 @@ Feature: cluster log forwarder features
     And evaluation of `JSON.parse(@result[:stdout])['count']` is stored in the :app_log_count_2 clipboard
     And the expression should be true> cb.app_log_count_2 > 0
     """
+  # @author kbharti@redhat.com
+  # @case_id OCP-39786
+  @admin
+  @destructive
+  Scenario: Send logs to both external fluentd and internalES
+    #Creating secure fluentd receiver
+    Given I switch to the first user
+    And I have a project
+    And evaluation of `project` is stored in the :fluentd_proj clipboard
+    Given fluentd receiver is deployed as secure with mTLS enabled in the "<%= cb.fluentd_proj.name %>" project
+    # Creating app
+    And I create a project with non-leading digit name
+    And evaluation of `project` is stored in the :proj clipboard
+    And I obtain test data file "logging/loggen/container_json_log_template.json"
+    When I run the :new_app client command with:
+      | file | container_json_log_template.json |
+    Then the step should succeed
+    #Creating Cluster Logging Forwarder instance
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-logging" project
+    Given admin ensures "instance" cluster_log_forwarder is deleted from the "openshift-logging" project after scenario
+    And I obtain test data file "logging/clusterlogforwarder/fluentd/secure/clusterlogforwarder.yaml"
+    When I process and create:
+      | f | clusterlogforwarder.yaml |
+      | p | URL=tcp://fluentdserver.<%= cb.fluentd_proj.name %>.svc:24224 |
+    Then the step should succeed
+    And I wait for the "instance" cluster_log_forwarder to appear
```

Comment:
I read [OCP-39786](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-39786), it says `Note:  Be sure there isn't clusterlogforwarder  --> oc delete clusterlogforwarder instance` in step 2, this case doesn't test the CLF, so no need to create clusterlogforwarder here.

Reply:
@kabirbhartiRH  yes, that is a legacy log forward method.  We have to create configmap  named secure-forward and secret named secure-forward(step 3).   The damonset fluentd can mount these two resource(step 4). The fluentd can use these source to sent logs to external fluentd and defaut ES(step 5, step 6).

Reply:
@QiaolingTang @anpingli I have made the changes now to exclude the CLF and use the legacy forward method of secrets and configmap. Please review